cdap-e2e-tests/src/e2e-test/java/io/cdap/cdap/controlcenter/stepsdesign/TestSetupHooks.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.controlcenter.stepsdesign;
+
+import com.google.cloud.bigquery.BigQueryException;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+public class TestSetupHooks {
+  public static String bqTargetTable = StringUtils.EMPTY;
+  public static String bqSourceTable = StringUtils.EMPTY;
+  public static String datasetName = PluginPropertyUtils.pluginProp("dataset");
+
+  @Before(order=1 ,value = "@BQ_INSERT_INT_SOURCE_TEST")
```

Comment:
Add the after hook for this before hook.

Reply:
added the hook